tests/unit/Domain/Package/PackageTest.php

```diff
@@ -0,0 +1,98 @@
+<?php 
+declare(strict_types = 1);
+
+namespace PackageHealth\PHP\Test\Unit\Domain\Package;
+
+use PHPUnit\Framework\TestCase;
+
+use PackageHealth\PHP\Domain\Package\Package;
+
+final class PackageTest extends TestCase
+{
+  private Package $package;
+  private \DateTimeImmutable $createdAt;
```

Comment:
Prefer adding `use DateTimeImmutable` to the file header and removing the leading `\`